### ISSUE TITLE
JzzMidiInput erroring when MidiInput messageRecieved to listener 

### ISSUE THREAD

**Issue by TrevorSStone** [atsushieno/ktmidi#47]
Trying out the library for the browser and I ran into an issue. When using JzzMidiInput and a registered `OnMidiReceivedEventListener` is going to be triggered the library calls 

```kotlin    
private val outFunc : dynamic = { midi : dynamic ->
        val arr = midi.data
        this.listener?.onEventReceived(arr, 0, arr.length, 0)
    }
```

For some reason `midi.data` is undefined in both Firefox and Chrome on my Mac. I was able to locally just change the `midi.data` to `midi` and it solved the issue. I'm happy to put up the minor PR, but I just wanted to check if I was missing something before I do it

**Comment by atsushieno** [comment]
Whoa, I did not notice that. Thanks for the report! You didn't miss anything, the fact is even funkier that I left it in the middle after I could compile it after I noticed that my ktmidi app does not work w/ JS yet. If you could send a PR that would be awesome. If it's going to be annoying I will fix it.

**Comment by atsushieno** [comment]
This should be fixed by 787994b (though unverified; some tests and/or example code should be added after v0.6 work).
